lib/modules/measure/MeasureExporter.ts

```diff
@@ -224,6 +229,66 @@ export class MeasureExporter {
     }
   }
 
+  private async generateMeasureColumns(
+    documentMeasures: NamedMeasures
+  ): Promise<Array<Column & { isMeasure: true }>> {
+    /**
+     * @example
+     * {
+     *   temperature: ['temperature.type'],
+     *   acceleration: [
+     *     'acceleration.properties.x.type',
+     *     'acceleration.properties.y.type',
+     *     'acceleration.properties.z.type',
+     *     'accuracy.type'
+     *   ]
+     * }
+     */
+    const mappingsByMeasureType: Record<string, string[]> = {};
+    const measures: Array<Column & { isMeasure: true }> = [];
+
+    for (const { name, type } of documentMeasures) {
+      if (!(type in mappingsByMeasureType)) {
+        const { measure } = await ask<AskModelMeasureGet>(
+          "ask:device-manager:model:measure:get",
+          { type }
+        );
+
+        mappingsByMeasureType[type] = Object.keys(
+          flattenObject(measure.valuesMappings)
+        );
+      }
+
+      for (const measure of mappingsByMeasureType[type]) {
+        /**
+         * @example
+         * 'temperature', 'acceleration.x', 'accuracy'
+         */
+        const path = measure.replace(".type", "").replace(".properties", "");
+        /**
+         * @example
+         * '', 'x', 'accuracy'
+         */
+        const pathWithoutType = path.replace(new RegExp(type + "[.]?"), "");
+
+        /**
+         * @example
+         * 'temperature', 'acceleration.x', 'acceleration.accuracy'
+         */
+        const header = `${name}${
+          pathWithoutType !== "" ? `.${pathWithoutType}` : ""
+        }`;
```

Comment:
```suggestion
        const header = `${name}.${pathWithoutType}`;
```

This ternary condition seems to be useless.
Unless if you want to handle nullish value in this case you have an error and the better solution is to use nullish coalsing, like `pathWithoutType ?? ''`

Reply:
You're not entirely right, it **was** useful, but your comment made me rework this crap. When I did it, I did not know how to do it in a more proper way.

I've fixed it. I know use a simple regex to do what I wanted to do.
I will add a comment in this PR explaining what I did